policyengine_us/variables/gov/states/nj/tax/income/nj_income_tax_before_refundable_credits.py

```diff
@@ -8,3 +8,9 @@ class nj_income_tax_before_refundable_credits(Variable):
     unit = USD
     definition_period = YEAR
     defined_for = StateCode.NJ
+
+    def formula(tax_unit, period, parameters):
+        main_income_tax = tax_unit("nj_main_income_tax", period)
```

Comment:
out of direct scope for this pr, but while you're in here could you rename this `nj_income_tax_before_credits` for consistency with other states?

Reply:
I was trying to match NY which has `ny_main_income_tax` and then `ny_income_tax_before_credits`, which adds the main tax and the supplemental tax. Since NJ doesn't have a supplemental tax, I was using the main tax directly. Should I change NJ main tax to `nj_income_tax_before_credits` or create a variable `nj_income_tax_before_credits` that just adds `nj_main_income_tax` for now to more closely match the NY structure?